api/envoy/config/resource_monitor/injected_resource/v2alpha/injected_resource.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package envoy.config.resource_monitor.injected_resource.v2alpha;
+option go_package = "v2alpha";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Injected resource]
+
+// The injected resource monitor allows injecting a synthetic resource pressure into Envoy
+// via a text file, which must contain a floating-point number in the range [0..1] representing
+// the resource pressure and be updated atomically by a symbolic link swap.
+// This is intended primarily for integration tests to force Envoy into an overloaded state.
```

Comment:
Are there any non-test use cases for this? If there are (and even really if there aren't) I'd think we might want to make the injected resource config a oneof a filename, or the value to use. That would be more flexible.